src/neptune/handler.py

```diff
@@ -663,6 +664,14 @@ def fetch_files_list(self) -> List[ArtifactFileData]:
         """
         return self._pass_call_to_attr(function_name="fetch_files_list")
 
+    def fetch_fileset_files(self) -> List[FileEntry]:
```

Comment:
We already have `list_artifact_files`. Isin't the listing more proper naming here?